lib/internal/streams/utils.js

```diff
@@ -8,7 +8,7 @@ const {
 } = primordials;
 
 // We need to use SymbolFor to make these globally available
-// for interopt with readable-stream, i.e. readable-stream
+// for interoperate with readable-stream, i.e. readable-stream
```

Comment:
```suggestion
// for interoperability with readable-stream, i.e. readable-stream
```

Reply:
@lpinca Thank you for your suggestion! I fixed, please check it 😄